wspace/workspace.go

```diff
@@ -62,3 +65,33 @@ func Find(dir string) (string, error) {
 	}
 	return Find(filepath.Dir(dir))
 }
+
+// FindRepoBuildFiles parses the WORKSPACE to find BUILD files for non-Bazel
+// external repositories, specifically those defined by one of these rules:
+//   new_local_repository(), new_git_repository(), new_http_archive()
+func FindRepoBuildFiles(root string) (map[string]string, error) {
```

Comment:
seems fine, but where will this be used?

Reply:
Check out my next change here: https://github.com/drigz/buildtools/commit/635278c78f52adca6bda6d34f5746cc2b4c44c88

I've been using it for running buildozer against BUILD files for non-bazel repositories.